vendor/assets/javascripts/mediaelement/mediaelement-and-player.js

```diff
@@ -8068,12 +8068,14 @@ function secondsToTimeCode(time) {
 			seconds = (f / timeBase % 60).toFixed(secondsDecimalLength);
 		}
 	} else {
-		hours = Math.floor(time / 3600) % 24;
-		minutes = Math.floor(time / 60) % 60;
+		var sec_num = time.toFixed(secondsDecimalLength);
+		hours   = Math.floor(sec_num / 3600);
```

Comment:
Was the `% 24` meant to be removed from this line?

Reply:
I missed it. Thanks!